build/app_template_files/__NativeTemplateAppFiles__/Podfile

```diff
@@ -0,0 +1,9 @@
+# Uncomment this line to define a global platform for your project
+# platform :ios, '6.0'
+
+target '__NativeTemplateAppName__' do
+
+pod 'SalesforceMobileSDK-iOS', '~> 3.3'
```

Comment:
Can we move ```3.3``` to a variable here and add a comment, so that we remember to update it every time we release?